test/unit/memory/load/tuple.cpp

```diff
@@ -45,13 +45,13 @@ TTS_CASE_TPL( "Check load behavior with soa_ptr", eve::test::scalar::all_types)
                               , reference.storage()
                               )};
 
-  auto [data0,idx0] = page<std::int8_t  , typename w8_t::cardinal_type >();
-  auto [data1,idx1] = page<T            , typename w8_t::cardinal_type >();
-  auto [data2,idx2] = page<double       , typename w8_t::cardinal_type >();
+  auto [data0,idx0] = page<std::int8_t  , typename w_t::cardinal_type >();
+  auto [data1,idx1] = page<T            , typename w_t::cardinal_type >();
+  auto [data2,idx2] = page<double       , typename w_t::cardinal_type >();
 
-  auto src = eve::soa_ptr    ( eve::as_aligned(&data0[idx0],typename w8_t::cardinal_type{})
+  auto src = eve::soa_ptr    ( eve::as_aligned(&data0[idx0],typename w_t::cardinal_type{})
                               , &data1[idx1] - 1
-                              , eve::as_aligned(&data2[idx2],typename w8_t::cardinal_type{})
```

Comment:
the test was correct. It's testing fixed size wide

Reply:
The problem here was as wide type, associated with data1, will have 16 elements, so during load it will load more than needed, changed. Current version increases fixed wide size.

Reply:
I don't understand you. w8_t is 8 elements. This verifies that you can load 8 elements. Not 16. 

What fails and how.